templates/github/.github/workflows/scripts/before_install.sh.j2

```diff
@@ -158,7 +158,7 @@ pip install docker netaddr boto3 ansible
 
 for i in {1..3}
 do
-  ansible-galaxy collection install amazon.aws && s=0 && break || s=$? && sleep 3
+  ansible-galaxy collection install "amazon.aws:1.5.0" && s=0 && break || s=$? && sleep 3
```

Comment:
it seems they reverted your fix!

Reply:
Damn it we are back where we started?

Reply:
not sure, but I believe so

Reply:
I don't see it reverted. This may actually be a new bug.